collect_app/src/test/java/org/odk/collect/android/projects/ProjectListItemViewTest.kt

```diff
@@ -0,0 +1,36 @@
+package org.odk.collect.android.projects
+
+import android.content.Context
+import android.graphics.Color
+import android.graphics.drawable.GradientDrawable
+import android.widget.TextView
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.equalTo
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.odk.collect.android.R
+
+@RunWith(AndroidJUnit4::class)
+class ProjectListItemViewTest {
```

Comment:
I'm a big fan of this pattern of pulling out "presenter" style views that hold all the knowledge of how to display a domain object. It lets you keep presentation logic for that data out of places like the Activity/Fragment, allows you to write super precise tests at the view level while keeping fuzzy at the app level. It also means that as the minutia of the view itself changes we don't need to change our app level tests - they only need to be concerned with the headline items (I can select a project, I can create a project etc).